ui/app/scripts/controllers/blockersCtrl.js

```diff
@@ -172,6 +180,23 @@ angular.module('apollo')
           $scope.allServices = tempServices;
       });
 
+      apolloApiService.getAllStacks().then(function(response) {
+        response.data.forEach(function(stack) {
+            $scope.allStacks.push(stack);
```

Comment:
Using push function will place the element at the end of the array, and I believe you need an indexed array.